yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -805,8 +495,7 @@ module ietf-yang-revisions {
   // RFC Ed.: replace XXXX (inc above) with actual RFC number and
   // remove this note.
 
-  revision 2022-11-29 {
-    rev:label "1.0.0-draft-ietf-netmod-yang-module-versioning-08";
+  revision 2024-02-19 {
```

Comment:
Heh.  So here's a question.  This module doesn't use YANG Semver to create a version.  And doing so would make Semver normative.  However, since this flies in the face of Semver's recommendation, what do we do here?  Nothing and assume these modules will be versioned later?

Reply:
Joe, I'm glad you noticed that. I wanted to bring this up and forgot. From a technical pov, I think we should add back ys:version and yes that means having yang-semver as normative reference. From a process pov, not sure what's the best way forward.
@rgwilton @jsterne

Reply:
Discussed in Feb 27th meeting and decision is to keep it as is.

Reply:
Closing this as per Feb 27th meeting.